functional_requirements/requirements.txt

```diff
@@ -1,111 +1,188 @@
-User Inputs:
+PART 1 User Interaction
+
+REQ 1.1 Input Buttons
+
+The device shall provide two general purpose input buttons for interacting with the user. 
 
 - button (A)
 - button (B)
 
-The device shall power on when button A is pressed for more than 3 seconds
-The initial display state upon power-on shall be clock
+REQ 1.2 Display
 
-The clock shall deactivate when buttons A+B are pressed for more than 3 seconds
+The device shall communicate with the user thought the output display.
 
-At power-on the clock shall read the settings and load the clock time from mobile phone. If connection is not available, default 00:00 is used.
+REQ 1.3 Activation
 
+The device shall power on when button A is pressed for more than 3 seconds.
 
-Display - views
+REQ 1.3 Deactivation
 
-- Clock with hands: default view at start-up
+The clock shall power off when buttons A+B are pressed for more than 3 seconds in any moment.

```

Comment:
Same as my comment on your other  pull request - state diagrams do not match "in any moment" - just from display selection menu.

https://github.com/markroche92/sw-eng-smart-watch/blob/master/uml/state_diagrams/display.png

Shall I try to update the state diagrams ?

Reply:
Just the settings diagram, the other are not supposed to illustrate an interaction with the user

Reply:
I think you should remove the A+B deactivation of Display State Diagram as implicit in any of your diagrams

Reply:
Okay, I've removed that here https://github.com/markroche92/sw-eng-smart-watch/pull/18